lms/djangoapps/mobile_api/course_info/views.py

```diff
@@ -88,6 +88,15 @@ def list(self, request, course, *args, **kwargs):
                     course_id=course.id,
                     static_asset_path=course.static_asset_path
                 )
+                handouts_html = replace_jump_to_id_urls(
+                    handouts_html,
+                    course_id=course.id,
+                    jump_to_id_base_url="/courses/{}/jump_to_id/".format(course.id)
+                )
+                handouts_html = replace_course_urls(
+                    handouts_html,
+                    course_key=course.id
+                )
```

Comment:
There probably should be a fairly straightforward way to do that. The modules are coming from the same place, after all.